DigitalLearningSolutions.Web/appsettings.Production.json

```diff
@@ -1,12 +1,12 @@
-{
```

Comment:
Was there supposed to be a change here?

Reply:
There was a change at one point, but that was then reverted. The line endings changed to CRLF as part of it, and haven't been reverted. Might as well keep them in my opinion.